ui/src/lib/components/CoreServicesWidget/component.svelte

```diff
@@ -0,0 +1,135 @@
+<script lang="ts">
+  import type { V1Pod } from '@kubernetes/client-node'
+  import type { UDSPackageStatus } from '$features/k8s/types'
+  import { getColorForStatus } from '$lib/features/k8s/helpers'
+  import { type CoreServiceType } from '$lib/types'
+  import { resourceDescriptions } from '$lib/utils/descriptions'
+  import { Cube, Information } from 'carbon-icons-svelte'
+  import * as _ from 'lodash'
+
+  export let coreServices: CoreServiceType[] = []
+  export let pods: V1Pod[] = []
+
+  const coreServicesMapping: Record<string, string> = {
+    authservice: 'Authorization',
+    grafana: 'Monitoring',
+    istio: 'Service Mesh',
+    keycloak: 'Identity Access Management',
+    loki: 'Log Aggregation',
+    'metrics-server': 'Metrics',
+    neuvector: 'Container Security',
+    'prometheus-stack': 'Monitoring',
+    vector: 'Log Aggregation',
+    velero: 'Backup & Restore',
+    'uds-runtime': 'Frontend Views & Insights',
+  }
+
+  const coreServiceKeys = Object.keys(coreServicesMapping)
+  let hasNoCoreServices: boolean = false
+  let transformedCoreServiceList: { name: string; status: UDSPackageStatus }[] = []
+  let hasPolicyEngineOperator: boolean = false
+  let hasServiceMesh: boolean = false
+
+  $: hasNoCoreServices = coreServices.every((service) => !coreServiceKeys.includes(service.metadata.name))
```

Comment:
```suggestion
  $: hasNoCoreServices = Array.isArray(coreServices) && coreServices.every((service) => !coreServiceKeys.includes(service.metadata.name));
```
this along with the suggestion below fixed that issue i was seeing where the view menu's stopped working